client/src/components/Main.tsx

```diff
@@ -166,17 +167,21 @@ const Part: React.FC = () => {
       if (headlessApi.current && update && params) {
         set({ loading: true })
         try {
-          productOrSolidId.current = await update(
+          productOrSolidIds.current = await update(
             headlessApi.current,
-            productOrSolidId.current,
+            cad instanceof solid ? productOrSolidIds.current as number : productOrSolidIds.current as number[],
             params,
```

Comment:
Same as above; These type casts look redundant. Is it only for the sake of explicity?

Reply:
See also [55f2cea](https://github.com/awv-informatik/buerli-examples/pull/11/commits/55f2cea068c459a65bacba60cea0c4af3010c575)